modules/textures/src/libs/README.md

```diff
@@ -1,5 +1,7 @@
 # Basis decoder
 
+Version 1.12
+
```

Comment:
Any release notes we can link to? Figure out if any new features etc.

Reply:
I don't know previous version of libs we had in the repository.
Their release notes are in README.md: https://github.com/BinomialLLC/basis_universal/blob/v1.12/README.md#release-notes